app/src/main/java/com/es/findsoccerplayers/position/PositionClient.java

```diff
@@ -0,0 +1,105 @@
+package com.es.findsoccerplayers.position;
+
+import android.Manifest;
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Context;
+
+import android.content.IntentSender;
+import android.location.LocationManager;
+import android.util.Log;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.common.api.ApiException;
+import com.google.android.gms.common.api.ResolvableApiException;
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationCallback;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.location.LocationSettingsRequest;
+import com.google.android.gms.location.LocationSettingsResponse;
+import com.google.android.gms.location.LocationSettingsStatusCodes;
+import com.google.android.gms.location.SettingsClient;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+
+import static android.content.ContentValues.TAG;
+
+public class PositionClient {
+
+    public static final int GPS_REQUEST = 1001;
+
+    //Location Request to build the Location Callback.
+    public static LocationRequest getLocationReq() {
+        LocationRequest locationRequest = new LocationRequest();
+        locationRequest.setInterval(10000);
+        locationRequest.setFastestInterval(5000);
+        locationRequest.setPriority(LocationRequest.PRIORITY_HIGH_ACCURACY);
+        return locationRequest;
+    }
+
+    public static void stopTrackingPosition(FusedLocationProviderClient fusedLocationProviderClient, LocationCallback locationCallback){
+        fusedLocationProviderClient.removeLocationUpdates(locationCallback);
+    }
+
+    public static void startTrackingPosition(FusedLocationProviderClient fusedLocationProviderClient, LocationCallback locationCallback){
+        fusedLocationProviderClient.requestLocationUpdates(getLocationReq(), locationCallback, null );
+    }
+
+    public static boolean isGpsOFF(Context context){
+        LocationManager locationManager = (LocationManager) context.getSystemService(Context.LOCATION_SERVICE);
+        if(!locationManager.isProviderEnabled(LocationManager.GPS_PROVIDER)){
+            return true;
+        }else {
+            return false;
+        }
+    }
+
+    public static void turnGPSon(final Context context){
```

Comment:
non capisco perché c'è un metodo uguale in GPSUtils. Magari se riesci a rendere anche più chiaro cosa fa questo codice meglio, perché onestamente non capisco senza commenti

Reply:
GpsUtils era da cancellare ma mi sono dimenticato. Infatto non lo uso mai. Aggiungo i commenti